### PR TITLE
Add the ability to edit user placed markers

### PR DESCRIPTION
**Pull request by Timi007**
**When merged this pull request will:**
- Add the ability to edit user placed markers by double clicking at a marker
- Remove the direct channel as a valid channel because selecting it ingame will not select the default (group) channel but instead the global
- Optimize the method to get the current map display
- To some extend also allow to copy markers by e.g. starting to edit a marker, canceling it and then creating a new marker with the same properties as the edited marker

### CODE REVIEW

**Comment by jonpas** [comment]
I like this! Editing markers!

**Comment by commy2** [comment]
Nice, this is why I love open source.

File: addons/markers/functions/fnc_getEnabledChannels.sqf [hunk 1]
```diff
@@ -40,12 +40,8 @@ if (_localize) then {
...
-
-    if (setCurrentChannel 5) then {
-        _enabledChannels pushBack localize "str_channel_direct";
-    };
 } else {
-    for "_i" from 0 to 5 do {
+    for "_i" from 0 to 4 do {
         if (setCurrentChannel _i) then {
             _enabledChannels pushBack _i;
         };
```
Comment (commy2) [inline addons/markers/functions/fnc_getEnabledChannels.sqf]: If I understand correctly, this means that using this getter while in direct channel, the channel gets changed to global (or some other one), which would violate the no side effects principle of a getter.

**Comment by Timi007** [comment]
@bux Corrected all spelling mistakes (at least those which I could find)

**Pull request merged**